### DISCUSSION TITLE
Implement !mp commands

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#16392] [Ideas]
I know a couple of people (including myself) are entertaining the idea of hosting a lazer-only tournament, and this would be a major help for that.

**Comment by smoogipoo** [comment]
For the immediate future we're hoping to bring as much implementation into the client itself, with things like automation of tournaments. Mp commands would be good to have though, and I've expressed this to @peppy, but will require osu-web implementation.
